internal/core/services/revocation.go

```diff
@@ -106,6 +115,73 @@ func convertCredentialStatus(credStatus interface{}) (verifiable.CredentialStatu
 	return verifiable.CredentialStatus{}, errors.New("failed cast credential status to verifiable.CredentialStatus")
 }
 
+func getRevocationStatusFromAgent(ctx context.Context, from, to, endpoint string, nonce uint64) (*verifiable.RevocationStatus, error) {
```

Comment:
you should use this resolver
https://github.com/iden3/iden3comm/blob/main/resolvers/agent.go

as well as you can switch to Credential Registry concept here https://github.com/iden3/go-schema-processor/blob/adbd0a5aa31811583d88aa9f48ff00b4bb1efd9f/verifiable/credential_status.go#L33

Reply:
this was deleted because It was not used